src/feinstaub-api.js

```diff
@@ -10,30 +12,35 @@ let api = {
   fetchNow() {
     return fetch(URL).then(response => response.json());
   },
-
   // fetches from /now, ignores non-finedust sensors
   // /now returns data from last 5 minutes, so we group all data by sensorId
   // and compute a mean to get distinct values per sensor
   getAllSensors() {
     return api.fetchNow().then(json => {
       let cells = _.chain(json)
+        .filter(json => json.node_moved == false)
         .map((value, key) => {
           let id = function(id) {
             for (let i in value.stats) {
-              return Number(value.stats[i].sensor_id);
+              if (value.node_moved == false) {
```

Comment:
This might be unnecessary since you already filtered [here](https://github.com/CodeForAfrica/sensors.AFRICA-AQ-map/pull/13/files#diff-3c9132b84184fb2fe1dc7c17226145dfR21)

Reply:
Done